src/react-reusable.js

```diff
@@ -57,8 +57,23 @@ export const useReuse = (fn, selector = identity, areEqual = shallowEqual) => {
   return localCopy;
 }
 
-export const reusable = (fn, options) => {
-  getStore().createUnit(fn, options);
+export const reusable = (fn) => {
+  getStore().createUnit(fn);
 
   return (selector, areEqual) => useReuse(fn, selector, areEqual);
-}
\ No newline at end of file
+}
+
+// TBD:
+// export const reusableReducer = (reducer, initialValue, init, options) => {
```

Comment:
why do we need reusableReducer and reusableState? I thought we could skip those by wrapping reusable on a custom hook.
Are these for situations where you don't want to write a custom hook?

Reply:
Just thinking that it might be useful for people who use reducers a lot. Just saves boilerplate, where you can write:  
```javasacript
const useUser = reusableReducer(reducer, initialState)
```
Instead of:  
```javasacript
const useUser = reusable(() => useReducer(reducer, initialState))
```